src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/haflow/update/actions/OnFinishedAction.java

```diff
@@ -86,7 +88,7 @@ private void updateFlowMonitoring(HaFlowUpdateFsm stateMachine) {
             // setup new if it is not single
             //TODO: Review logic during https://github.com/telstra/open-kilda/issues/5208
             if (targetNotSingle && (srcUpdated || dstUpdated)) {
-                stateMachine.getCarrier().sendActivateFlowMonitoring(null);
+                //stateMachine.getCarrier().sendActivateFlowMonitoring(null);
```

Comment:
Why did you comment this?

Reply:
To avoid another error. 
At org/openkilda/wfm/topology/flowhs/bolts/HaFlowUpdateHubBolt.java:244 we expect non-null object.